pom.xml

```diff
@@ -909,6 +958,18 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
I think it's the right thing to do; we'll have to see if excluding Jersey 1 breaks other things internal to Hadoop that we need that use Jersey 1. Looks like you caught one example for YARN testing.